--- a/src/edk2/BaseTools/Conf/tools_def.template
+++ b/src/edk2/BaseTools/Conf/tools_def.template
@@ -141,7 +141,7 @@
 DEFINE UNIX_IASL_BIN           = ENV(IASL_PREFIX)iasl
 DEFINE WIN_IASL_BIN            = ENV(IASL_PREFIX)iasl.exe
 
-DEFINE IASL_FLAGS              =
+DEFINE IASL_FLAGS              = -vw 6084
 DEFINE IASL_OUTFLAGS           = -p
 
 DEFINE DEFAULT_WIN_ASL_BIN      = DEF(WIN_IASL_BIN)
